test/slow/api_spec_slow.js

```diff
@@ -156,166 +158,224 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
+      });
 
-      await forge.package({ dir });
-    });
+      it('can make from custom outDir without errors', async () => {
+        const packageJSON = await readPackageJSON(dir);
+        packageJSON.config.forge.make_targets[process.platform] = ['zip'];
+        await fs.writeFile(path.resolve(dir, 'package.json'), JSON.stringify(packageJSON));
 
-    describe('after package', () => {
-      let resourcesPath = 'Test App.app/Contents/Resources';
-      if (process.platform !== 'darwin') {
-        resourcesPath = 'resources';
-      }
+        await forge.make({ dir, skipPackage: true, outDir: `${dir}/foo` });
 
-      it('should have deleted the forge config from the packaged app', async () => {
-        const cleanPackageJSON = JSON.parse(asar.extractFile(
-          path.resolve(dir, 'out', `Test App-${process.platform}-${process.arch}`, resourcesPath, 'app.asar'),
-          'package.json'
-        ));
-        expect(cleanPackageJSON).to.not.have.nested.property('config.forge');
+        // Cleanup here to ensure things dont break in the make tests
+        await fs.remove(path.resolve(dir, 'foo'));
+        await fs.remove(path.resolve(dir, 'out'));
       });
 
-      it('should not affect the actual forge config', async () => {
-        const normalPackageJSON = await readPackageJSON(dir);
-        expect(normalPackageJSON).to.have.nested.property('config.forge');
+      it('can package without errors with native pre-gyp deps installed', async () => {
+        await installDeps(dir, ['ref']);
+        await forge.package({ dir });
+        await fs.remove(path.resolve(dir, 'node_modules/ref'));
       });
 
-      function getMakers(platform) {
-        return fs.readdirSync(path.resolve(__dirname, `../../src/makers/${platform}`)).map(file => path.parse(file).name).filter(name => name !== 'flatpak');
-      }
+      it('can package without errors', async () => {
+        const packageJSON = await readPackageJSON(dir);
+        packageJSON.config.forge.electronPackagerConfig.asar = true;
+        await fs.writeFile(path.resolve(dir, 'package.json'), JSON.stringify(packageJSON, null, 2));
 
-      const goodMakers = [...getMakers(process.platform), ...getMakers('generic')];
-      const badPlatforms = ['darwin', 'linux', 'win32'].filter(p => p !== process.platform);
-      const badMakers = [];
-      badPlatforms.forEach(platform => badMakers.push(...getMakers(platform)));
-
-      const testMakeTarget = function testMakeTarget(target, shouldPass, ...options) {
-        describe(`make (with target=${target})`, async () => {
-          before(async () => {
-            const packageJSON = await readPackageJSON(dir);
-            packageJSON.config.forge.make_targets[process.platform] = [target];
-            await fs.writeFile(path.resolve(dir, 'package.json'), JSON.stringify(packageJSON));
-          });
+        await forge.package({ dir });
+      });
+
+      describe('after package', () => {
```

Comment:
I'm not sure we want to run the makers for import too... they take up the most time to run, and the testsuite already takes upwards of a half hour (for one worker).

Reply:
Sure, I left that in a separate commit in case you didn't think it was valuable. I can just rebase that commit away and leave only the other one. Or do you think a subset of the `after init` tests should be run for `import` rather than all or none?

Reply:
I don't think it's particularly valuable right now, although I'm sure I'm going to eat my words when we find some other import-related bug.

Reply:
Okay, I'll just delete that last commit then, and we can bring back the post-import tests (or a subset of them) if we find an area that seems particularly risky or regression-prone.